src/components/OperatorButtonContainer.jsx

```diff
@@ -0,0 +1,28 @@
+function OperatorButtonContainer({ updateOperation }) {
+  const onClickOperationButton = (e) => {
+    const newOperation = e.target.value;
+    updateOperation(newOperation);
+  };
+
+  return (
+    <div className="operations subgrid" onClick={onClickOperationButton}>
+      <button className="operation" value="/">
+        /
+      </button>
+      <button className="operation" value="x">
+        X
+      </button>
+      <button className="operation" value="-">
+        -
+      </button>
+      <button className="operation" value="+">
+        +
+      </button>
+      <button className="operation" value="=">
+        =
+      </button>
```

Comment:
> 지난 공통 피드백에서 button에 dataset을 주는 것이 리액트의 방식과는 거리가 멀다는 피드백이 있어서, value를 사용했는데, 이것도 dataset 사용과 비슷하게 리액트의 방식과는 다르지 않은가... 고민이 됩니다. ㅠㅠ
button을 Button과 같이 컴포넌트로 한 번 감싸 만들고, 이 컴포넌트에 props로 value에 해당하는 값을 주는 방식으로 만들면 조금 더 리액트스러울까요?

요 부분 질문 맞죠?! 저였다면 아래와 같이 구현했을 것 같아요!
```suggestion
     {OPERATIONS.map((operation, index) => (
        <button
          className="operation"
          onClick={() => onClickOperationButton(operation)}
        >
          {operation}
        </button>
      ))}
```

Reply:
좋아요~ 말씀해주신 방식으로 반영해봤습니다!! [`51cc483`](https://github.com/woowacourse/react-calculator/pull/56/commits/51cc4835722353059d5558d645f894048b7e7a39)